src/config.h

```diff
@@ -27,7 +28,7 @@ typedef enum {
 	Config_END_MARKER               = 10
 } Config_Option_Field;
 
-typedef void (*Config_on_change)(Config_Option_Field type); // callback function which being called when config param changed
+typedef bool (*Config_on_change)(Config_Option_Field type); // callback function which being called when config param changed
```

Comment:
why the change from `void` to `bool` ?

Reply:
Because the handler calling 				if(!Config_Option_get(type, &max_queued_queries)) {
which might return error and I thought it should be propagated.